scripts/release.star

```diff
@@ -35,10 +35,8 @@ def release_npm_packages_step(edition, ver_mode):
     if edition == 'enterprise':
         return None
 
-    commands = [
-        './node_modules/.bin/lerna bootstrap',
-        'echo "//registry.npmjs.org/:_authToken=$${NPM_TOKEN}" >> ~/.npmrc',
-    ]
+    commands = []
```

Comment:
I'm not sure about this. It feels strange that we have a step that releases the packages but only adds the release-packages.sh to the list of commands if `ver_mode = 'release'`. I'm likely missing something related to releases.